README.md

```diff
@@ -1,27 +1,103 @@
 # Family Promise of Spokane Data Sharing API
-
 ## Project Overview
-Problem: What happened to local utilities customers that fell behind on payments prior to ceasing to be customers? Many become homeless. How do we use our resources to prevent future homelessness?
+What happens to local utilities customers that fell behind on payments prior to ceasing to be customers? How can we use our resources to prevent future homelessness?
 
-Solution: An API that uses unique identifiers to determine whether an ex-customer began receiving services from service providers assisting people experiencing homelessness.
+Our Solution: Design an API that uses unique identifiers to determine whether an ex-customer began receiving services from service providers assisting people experiencing homelessness.
 
 ## Tech Stack
+Services: AWS, Docker, Jupyter Notebooks, Postman
+
+Languages: Python, SQL
+
+Backend: AWS API Gateway, AWS Lambda, ElephantSQL
+
+## Getting Started
+### Prerequisites
+- SQL
+- AWS Lambda and AWS API Gateway
+- PostgreSQL
+
+### Deployment to AWS
+#### Developer environment
+Build Amazon Linux image with Python 3.7 and pip
+
+```docker build -t example_image_name .```
+
+#### Installing dependencies
+
+All dependencies are already installed, but if for some reason you needed to delete and reinstall:
 
-Python, Flask, SQLAlchemy, AWS, Docker, elephantsql
+```docker run -v $(pwd):/aws -ti example_image_name```
+
+then
+
+```pip install bcrypt aws-psycopg2 pandas -t /aws```
+
+Do not install if these packages already exist in the aws folder.
+
+#### Packaging Lambda Function
+```zip -r example_filename.zip *```
+
+At this point you'll want to head over the AWS GUI for function creation at AWS Lambda. 
 
 ## Development
+### Architecture
+<img src="https://github.com/nonprofit-intake/family_promise_data_sharing/blob/dev/images/fampromarch.png" width="500" height="350">
+Backend deployed serverlessly through AWS API Gateway and AWS Lambda.
 
-how-to: pipenv, requirements.txt, env variables
+### Endpoint - Return User Info
 
-## Resources
+**URL**
 
-### Endpoint 1
+https://3yk0fzdvdh.execute-api.us-east-1.amazonaws.com/default/return_user_info
 
-#### Request
+**Description**
 
-    GET 
+Returns the last name, first name, enroll date, exit date, income at entry, income at exit, and exit destination of a user in database.
 
-#### Response
-```json
+**POST Request**
```

Comment:
Example request and response JSON objects are cut off when you look at document in markdown. Try adding json or yaml after the triple back ticks, see: 

https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki

Reply:
That makes sense, someday I will be the master of markdown.